base_layer/core/src/transactions/transaction_protocol/proto/transaction_sender.proto

```diff
@@ -17,6 +17,8 @@ message SingleRoundSenderData {
     TransactionMetadata metadata = 5;
     // Plain text message to receiver
     string message = 6;
+    // Allow fee greater than amount
```

Comment:
These feel like two different things (allowing fee > amount and dust collection). Specifically, preventing dust collection is typically a special case of the former, so rather just name and comment using the former.

Reply:
Is github being github again?

I'm still seeing `dust_collector_enabled` when `allow_fee_gt_amount` was the agreed new name

Reply:
Sorry, have not pushed at the time yet.